README.md

```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## Shape Generators
 
+ - [SHACL Play!](https://shacl-play.sparna.fr/play/convert) - This utility applies a set of SHACL rules to convert an RDF dataset into another structure. Some SHACL rules are provided in the catalog to convert OWL ontologies to SHACL. 
```

Comment:
- change name to "SHACL Play! Convertor"
- Shorten descr to "Converts between RDF schema formats. Has OWL-to-SHACL (open, semi-closed, closed)"